sensor_msgs/Image image
float32 left
float32 top
float32 right
float32 bottom
uint32 class_id
float32 probability

